web-services-app/src/test/java/edu/unc/lib/boxc/web/services/rest/DatastreamRestControllerIT.java

```diff
@@ -105,13 +119,15 @@ public class DatastreamRestControllerIT extends AbstractAPIIT {
     @BeforeEach
     public void initLocal() {
         closeable = openMocks(this);
+        connectionManager = new PoolingHttpClientConnectionManager();
```

Comment:
This doesn't seem to be used other than to shut it down

Reply:
oh is that not for Wiremock?

Reply:
I don't think so, but it's worth trying to remove it and seeing if the test passes